erdgo/go.mod

```diff
@@ -5,5 +5,8 @@ go 1.13
 require (
 	github.com/ElrondNetwork/elrond-go v1.1.4
```

Comment:
maybe you can update the version of elrond-go

Reply:
Done